pinot-broker/src/test/java/org/apache/pinot/broker/requesthandler/LiteralOnlyBrokerRequestTest.java

```diff
@@ -177,4 +177,55 @@ public void testBrokerRequestHandlerWithAsFunction()
     Assert.assertEquals(brokerResponse.getResultTable().getRows().get(0)[1], 1577836800000L);
     Assert.assertEquals(brokerResponse.getTotalDocs(), 0);
   }
+
+  /** Tests for EXPLAIN PLAN for literal only queries. */
+  @Test
+  public void testExplainPlanLiteralOnly()
+      throws Exception {
+    SingleConnectionBrokerRequestHandler requestHandler =
+        new SingleConnectionBrokerRequestHandler(new PinotConfiguration(), null, ACCESS_CONTROL_FACTORY, null, null,
+            new BrokerMetrics("", PinotMetricUtils.getPinotMetricsRegistry(), true, Collections.emptySet()), null);
+
+    // Test 1: select constant
+    JsonNode request = new ObjectMapper().readTree("{\"sql\":\"EXPLAIN PLAN FOR SELECT 1.5, 'test'\"}");
+    RequestStatistics requestStats = new RequestStatistics();
+    BrokerResponseNative brokerResponse = requestHandler.handleRequest(request, null, requestStats);
+
+    checkExplainResultSchema(brokerResponse.getResultTable().getDataSchema(),
+        new String[]{"Operator", "Operator_Id", "Parent_Id"},
+        new DataSchema.ColumnDataType[]{DataSchema.ColumnDataType.STRING, DataSchema.ColumnDataType.INT,
+            DataSchema.ColumnDataType.INT});
+
+    Assert.assertEquals(brokerResponse.getResultTable().getRows().size(), 1);
+    Assert.assertEquals(brokerResponse.getResultTable().getRows().get(0),
+        new Object[]{"SELECT(selectList:literal)", 0, -1});
```

Comment:
It should show 1.5 imo

Reply:
1.5 is also a literal, so same reason as last comment.